calliope/cli.py

```diff
@@ -56,6 +57,13 @@
     help='Filename to save profile to if enabled --profile.'
 )
 
+
+_fail_when_infeasible = click.option(
+    '--fail_when_infeasible/--no_fail_when_infeasible', is_flag=True, default=True,
```

Comment:
I followed Calliope's convention of using underscores in command line options. But is that something you consider changing at one point, maybe for a 1.0 release? ;-).

Reply:
I think this could be updated a bit earlier than a 1.0 release ;)